CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# dbt_zendesk_source v0.15.0
+
+## Under the Hood
+- (Affects Redshift only) Updates the `union_zendesk_connections` macro to use a limit 1 instead of limit 0 for empty tables.
+  - When a table is empty, Redshift ignores explicit data casts and will materialize every column as a `varchar`. Redshift users may experience errors in downstream transformations as a consequence.
+  - For each staging model, if the source table is not found, the package will create a empty table with 0 rows for non-Redshift warehouses and a table with 1 all-`null` row for Redshift destinations. The 1 row will ensure that Redshift will respect the package's datatype casts.
```

Comment:
```suggestion
  - For each staging model, if the source table is not found, the package will create a empty table with zero rows for non-Redshift warehouses and a table with one all-`null` row for Redshift destinations. The one row will ensure that Redshift will respect the package's datatype casts.
```

Reply:
Thanks Avinash! I didn't know that! I actually like the version I have currently because I think it's better readability. It also follows the same note in the recent [Qualtrics release](https://github.com/fivetran/dbt_qualtrics/releases), if we want to keep it standardized across packages.

Reply:
@fivetran-reneeli makes sense!